SUMO-0: Add an example of how to send a test trace to sumo to the HTTP traces docs (#1198)
SumoLogic/sumologic-documentation: docs/apm/traces/get-started-transaction-tracing/http-traces-source.md

Now, that you have created an HTTP Traces Source you can send a trace to it with
an HTTP POST request. The request's body should look like the following in the Zipkin v2 format
and you can simply send it to the source URL that you created in the previous step.

```
[
  {
    "id": {{parentSpanId}},
    "traceId": {{traceId}},
    "name": "metrics",
    "timestamp": {{timestamp0}},
    "duration": 2000000,
    "kind": "SERVER",
    "localEndpoint": {
      "serviceName": "test_service"
    },
    "tags": {
      "customTag": "tag"
    }
  },
  {
    "id": "352bff9a74ca9c1a",
    "traceId": {{traceId}},
    "parentId": {{parentSpanId}},
    "name": "metricsMethod",
    "timestamp": {{timestamp1}},
    "duration": 1000000,
    "kind": "SERVER",
    "localEndpoint": {
      "serviceName": "test_service2"
    },
    "tags": {
      "customTag": "tag"
    }
  },
  {
    "id": "352bff9a74ca9b1e",
    "traceId": {{traceId}},
    "parentId": {{parentSpanId}},
    "name": "metricsMethod2",
    "timestamp": {{timestamp2}},
    "duration": 1000000,
    "kind": "SERVER",
    "localEndpoint": {
      "serviceName": "test_service3"
    },
    "tags": {
      "customTag": "tag"
    }
  }
]
```

> In the example above we have three spans, one parent span without `parentId` and it's two children spans

Reviewer: ```suggestion
In the example above we have three spans, one parent span without `parentId` and its two children spans
```

> which refer to it's `id` with the `parentId` field.

Reviewer: ```suggestion
which refers to its `id` with the `parentId` field.
```

You can replace the variables `parentSpanId` and `traceId` with random hexadecimal ids 

> like e.g. `462bdf9a74ca9e2c` and `872cef9a74ca9c3a`. And `timestamps` with the current epoch timestamp

Reviewer: ```suggestion
like `462bdf9a74ca9e2c` and `872cef9a74ca9c3a`. You can replace `timestamps` with the current epoch timestamp
```

> in microseconds e.g. `1671096513000000`.

Reviewer: ```suggestion
in microseconds, for example `1671096513000000`.
```
